app/routes/bookmarks.tsx

```diff
@@ -36,3 +36,4 @@ export default function BookmarksLayout() {
 }
 
 export const ErrorBoundary = AppErrorBoundary;
+export const CatchBoundary = AppErrorBoundary;
```

Comment:
## Errors Step 3

All of our routes should also set the `CatchBoundary` to `AppErrorBoundary`. 

Once the dust settles on this starter, I'll make this route of the "route" vs code snippet.